src/test/java/com/github/tomakehurst/wiremock/StubbingAcceptanceTest.java

```diff
@@ -760,6 +758,86 @@ void matchingOnMultipartRequestBodyWithNotContaining() {
     assertThat(response.statusCode(), is(HTTP_OK));
   }
 
+  @Test
+  void matchingOnMultipartRequestBodyWithFilename() {
+    stubFor(
+        post(urlEqualTo("/match/this/part/filename"))
+            .withMultipartRequestBody(
+                aMultipart()
+                    .withHeader("Content-Type", containing("application/octet-stream"))
+                    .withFileName("plain-example.txt"))
+            .willReturn(aResponse().withStatus(HTTP_OK).withBodyFile("plain-example.txt")));
```

Comment:
I'm not sure there's any need to return anything in the body since you're just asserting that it matched via the response status.

This could be simplified (as could the other similar lines) to:
```java
.willReturn(ok())
```

Reply:
Agree, Thanks for the suggestion!